example/src/PickerAutoSuggestExample.js

```diff
@@ -51,71 +54,119 @@ class PickerAutoSuggestExample extends React.Component<{}, {}> {
           value: newValue,
         });
       },
-      onKeyDown: (e:any) => {
-        if ((e.keyCode === 8) && this.state.chipListState.length && !this.state.value.length) {
-          const yellowed = this.state.chipListState.slice(this.state.chipListState.length - 1);
-          if (yellowed[0]['markedForDelete']) {
-            const chipListState = this.state.chipListState.slice(0, this.state.chipListState.length - 1);
-            const language = this.state.chipListState.slice(this.state.chipListState.length - 1)[0];
-            delete language['text'];
-            language['markedForDelete'] = false;
-            const itemsList = this.state.itemsList.concat(language);
-            this.setState({
-              chipListState,
-              itemsList,
-            });
-          } else {
-            yellowed[0]['markedForDelete'] = true;
-
-            this.setState({
-              chipListState: this.state.chipListState,
-            });
-          }
-        } else if (this.state.chipListState.length && !this.state.value.length && !(e.keyCode === 38) && !(e.keyCode === 40)) {
-          if (this.state.chipListState[this.state.chipListState.length - 1]['markedForDelete']) {
-            this.state.chipListState[this.state.chipListState.length - 1]['markedForDelete'] = false;
-            this.setState({ chipListState: this.state.chipListState });
-          }
+      onfocus: (e:any) => {
```

Comment:
Move methods needed for the component to work internally and give the users limited or optional access.

Reply:
@harsimranb 
Do we need these changes? I guess PickerAutoSuggestExample is not related to Picker component at all. Because without having use of PickerAutoSuggestExample, we can use all features of Picker component. any thoughts on this?

Reply:
@HiirenP That's an interesting question.  I do think that this example has the people picker implementation we will be using.  You can check with @weegrom as well.